File: solutions/beecrowd/1197/1197.dart
import 'dart:io';

void main() {
  int v, t;
  var line;

  while ((line = stdin.readLineSync()) != null) {
    var numbers = line.split(' ');
    v = int.parse(numbers[0]);
    t = int.parse(numbers[1]);

    print(v * t * 2);
  }
}

File: solutions/beecrowd/1197/1197.ts
process.stdin.resume()
process.stdin.setEncoding('utf8')

let input = ''

process.stdin.on('data', (chunk: string) => {
  input += chunk
})

process.stdin.on('end', () => {
  const lines = input.trim().split('\n')

  lines.forEach(line => {
    const [v, t] = line.trim().split(' ').map(Number)
    console.log(v * t * 2 + 0) // + 0 to avoid -0 weird behavior
  })
})
